docs/plone_training_config/instructions.md

```diff
@@ -136,7 +136,7 @@ Now you can run the buildout for the first time:
 ./bin/buildout
 ```
 
-This will take **very long** time and produce a lot of output because it downloads and configures more than 260 Python packages. Once it is done you can start your Plone instance with
+This will take **a long time** (~10 minutes on the least powerful Linode) and will produce a lot of output because it downloads and configures more than 260 Python packages. Once it is done you can start your Plone instance with
```

Comment:
```suggestion
This will take **a long time** (~10 minutes on the least powerful Linode) and will produce a lot of output because it downloads and configures more than 260 Python packages.
Once it is done, you can start your Plone instance with the following command.
```